src/OpenLoco/src/Windows/IndustryList.cpp

```diff
@@ -449,6 +499,20 @@ namespace OpenLoco::Ui::Windows::IndustryList
                     auto point = Point(440, yPos);
                     drawingCtx.drawStringLeftClipped(rt, point, 138, Colour::black, StringIds::production_transported_percent, &args);
                 }
+                // Industry Production Last Month
+                if (industry->canProduceCargo())
```

Comment:
this is the same `if` as line 492, you can remove this one

Reply:
This is more readable this way imho, as every column has its own block. Else you either have more indenting or a less readable control flow.

Reply:
Fair, but someone else reading the code does not know that and will question why there are two identical blocks and wonder if one of them is wrong.

The way to do this is have a single if statement and scoped blocks, eg

```
if (industry->canProduceCargo())
{
    {
        // column 1 stuff
    } 
    {
        // column 2 stuff
    }
}
```

You can see this is used frequently in loco, even just in the lines above in this very method.

The other alternative is simply put both column pieces into separate methods, and then it becomes trivial:
```
if (industry->canProduceCargo())
{
    DoColumn1Stuff();
    DoColumn2Stuff();
}
```